recipe/meta.yaml

```diff
@@ -33,24 +33,28 @@ build:
     - copy lib\\{{ cuda_compiler_version }}\\cutensor.lib %LIBRARY_LIB%\\  # [win64 and cuda_compiler_version in ("10.1", "10.2")]
     - copy lib\\11\\cutensor.dll %LIBRARY_BIN%\\                           # [win64 and cuda_compiler_version in ("11.0", )]
     - copy lib\\11\\cutensor.lib %LIBRARY_LIB%\\                           # [win64 and cuda_compiler_version in ("11.0", )]
+  ignore_run_exports_from:
+    - {{ compiler('c') }}    # [linux]
+    - {{ compiler('cuda') }}
   run_exports:
     - {{ pin_subpackage('cutensor') }}
-    # The package built on CUDA 11.0 should work on any 11.x
-    - {{ pin_compatible('cudatoolkit') }}      # [cuda_compiler_version == "11.0"]
-    - cudatoolkit {{ cuda_compiler_version }}  # [cuda_compiler_version != "11.0"]
   missing_dso_whitelist:
     - '*/libgomp.*'  # [linux]
 
 requirements:
   build:
+    - {{ compiler('c') }}
+    - {{ compiler('cuda') }}
     - sysroot_linux-64 2.17  # [linux64]
     - libgomp  # [linux]
-    - cudatoolkit {{ cuda_compiler_version }}
+  host:
```

Comment:
However there was another issue, which is `cudatoolkit` was included in `build`. Really `build` should only be build tools used for their executables (`make`, compilers, etc.). So this likely was causing us issues

Reply:
I am sure I have tried Isuru's change earlier (apart from adding an empty host), and I think testing the dependency by running conda install in the test section is what's causing problem. I don't think it's a valid approach, likely involving how repodata is made seen by the solver.